test/memspaces/memspace_numa.cpp

```diff
@@ -6,6 +6,7 @@
 #include "memspace_fixtures.hpp"
 #include "memspace_helpers.hpp"
 #include "memspace_internal.h"
+#include "umf/memspace.h"
```

Comment:
nit: #include <umf/memspace.h> and move <> includes up

Reply:
Done